src/Collections-Strings/Symbol.class.st

```diff
@@ -495,6 +495,15 @@ Symbol >> isUnary [
 	^ self precedence = 1
 ]
 
+{ #category : #sorting }
+Symbol >> keywordsStrict [
+	"Returns the keywords of the provided selector. Does not assume that the reciever is keyword-based."
+
```

Comment:
reciever -> receiver

Please add also some executable comments.
I could not understand what is does not assume. So the examples will be more self explanatory